### DISCUSSION TITLE
possible new rules for this

### DISCUSSION THREAD

**Discussion by gengjiawen** [eslint/eslint#15545] [Ideas] [answered]
Maybe eslint can help with this kind of situations ?

https://github.com/nodejs/node/issues/41706

**Comment by aladdin-add** [comment]
Per our [rules policies](https://eslint.org/blog/2020/05/changes-to-rules-policies), we can only accept new rules if they relate to new ECMAScript features that reached stage 4 in the preceding 12 months, so this proposal doesn't qualify. You can always make a custom rule and publish a plugin.

**Comment by mdjermanovic** [comment, marked as answer]
I think this would be reported by `unicorn/no-array-callback-reference` rule:

https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-array-callback-reference.md
